content/docs/command-reference/gc.md

```diff
@@ -44,8 +44,29 @@ If the `--cloud` (`-c`) flag is used, this command deletes unused data from the
 to deleting it from the local DVC cache. To specify a DVC remote to delete from,
 use the `--remote` (`-r`) option.
 
-> ⚠️ Danger: cloud deletion is irreversible unless there is another DVC remote
-> or a manual backup with the same data.
+<admon type="warn">
+
+Cloud deletion is irreversible unless there is another DVC remote or a manual
+backup with the same data.
+
+</admon>
+
+### Cleaning shared cache or shared remote
+
+If a single remote or a single [cache is shared] among different projects and
+those projects have overlapped DVC-tracked files, removing files via `gc` in one
+project may break some links in another project.
```

Comment:
```suggestion
### Cleaning shared cache (or remote)

If a [cache is shared] among different projects that
track some of the same files, using `dvc gc` in one
project will break those overlapping data links in the other projects.
```

I'm removing "remote" because we don't really mention combining this with the `-r` option, which would further complicate the explanation. Hopefully people can infer this from the title.

Reply:
`shared remote` can be good for search

Reply:
may be share cache (or remote) is enough - not sure (I prefer usually explicit vs optimized wording)

Reply:
anyway, Jorge, please take it over - commit suggestions as you see and let's merge this. This kind of things we should be in 5 minutes ideally :)